sup3r/cli.py

```diff
@@ -28,26 +32,86 @@ def main(ctx, config_file, verbose):
     ctx.obj['VERBOSE'] = verbose
 
 
-@main.group()
+@main.group(invoke_without_command=True)
```

Comment:
You should look up click groups vs. commands. Any function like forward_pass that doesnt have any follow on commands should be the final command and not be a group. You only need these to be groups if there are follow on commands like the valid config keys thing

Reply:
@bnb32 one more thing on this, making the commands into commands instead of groups i dont think you need the lines that say `if ctx.invoked_subcommand is None:` because there wont be an option for subcommand.

Reply:
yeah good point